bundled/tool/install_viz_dependencies.py

```diff
@@ -18,6 +18,9 @@ def install_dependencies(extension_root_dir):
     try:
         import fastapi
         import orjson
+        import kedro_telemetry
+        from packaging.version import parse
```

Comment:
We can remove this right? we are already doing it in `bundled/tool/install_telemetry_dependencies.py`

Reply:
indeed - removed